contracts/bridge/L2ECOBridge.sol

```diff
@@ -227,16 +234,16 @@ contract L2ECOBridge is IL2ECOBridge, CrossDomainEnabledUpgradeable {
         bytes calldata _data
     ) internal {
         // Burn the withdrawn tokens from L2
-        l2EcoToken.burn(msg.sender, _amount);
+        l2Eco.burn(msg.sender, _amount);
 
         // Construct calldata for l1TokenBridge.finalizeERC20Withdrawal(_to, _amount)
-        address l1Token = l2EcoToken.l1Token();
+        address l1Token = l2Eco.l1Token();
```

Comment:
was l1Token() supposed to remain on the l2Eco contract?

Reply:
dont we already have this in l1Eco?

Reply:
good catch